fluss-client/src/test/java/com/alibaba/fluss/client/table/FlussTableITCase.java

```diff
@@ -101,6 +102,7 @@ void testAppendOnly() throws Exception {
 
     @ParameterizedTest
     @ValueSource(booleans = {true, false})
+    @Disabled("TODO, fix me in #116")
```

Comment:
Will this be fixed in followup PR?

Reply:
Yes, and there're still some unstable tests need to be fixed in followup pr